server/utils/oasysImportUtils.ts

```diff
@@ -117,8 +117,12 @@ export const findSummaryLabel = (questionNumber: string, summaries: Array<OASysQ
   let summary: OASysQuestion | undefined = summaries.find(i => i.questionNumber === questionNumber)
 
   if (!summary) {
+    const redactedSummaries = summaries.map(question => ({
+      ...question,
+      answer: question.answer ? '***redacted***' : undefined,
```

Comment:
As an empty string is falsey, this would return `undefined` rather than the empty string. I don't know if that matters at all?

Reply:
This is used in `JSON.stringify()`, so this results in the property not being included at all:

```js
JSON.stringify({ foo: undefined }) // returns '{}'
```

Reply:
Oh, I see what you mean, sorry -- if there is an answer but it is empty, then the property won't be added at all... I wouldn't worry, the main thing here is to show 'there was some content to the answer but it is redacted'. I don't think not having the property for an empty answer is a worry, in that respect!